preload.js

```diff
@@ -414,7 +414,8 @@ window.lokiFeatureFlags = {
   privateGroupChats: true,
   useSnodeProxy: !process.env.USE_STUBBED_NETWORK,
```

Comment:
Might be wise to set this to `false`? 
Once we have onion requests we can strip out all snode proxying code

Reply:
file server code can still use proxy where it can't use OR

Reply:
We should remove proxy code when we no longer use it, but I think the file server is not on onion requests yet.

Reply:
👍